mongodb/docs-worker-pool
# [DOP-3396]: Guides search index jobs are not enqueuing

## Ticket
[DOP-3396](https://jira.mongodb.org/browse/DOP-3396)

## Test Plan
1. Go to the [following build](https://workerpoolstaging-qgeyp.mongodbstitch.com/pages/job.html?collName=dotcom_queue&jobId=63cad941f604c938c792a809) and verify that the build is successful.
2.  Verify that the [search manifest job is enqueued](https://workerpoolstaging-qgeyp.mongodbstitch.com/pages/job.html?collName=dotcom_queue&jobId=63cadabb0ca3b75c00f89464)
3.  Verify that the [search manifest job is executed successfully](https://workerpoolstaging-qgeyp.mongodbstitch.com/pages/job.html?collName=dotcom_queue&jobId=63cadabb0ca3b75c00f89464)
